index.html

```diff
@@ -14,34 +14,52 @@
     />
   </head>
   <body class="bg-img">
-    <header class="header">
+    <header class="header mobile-app">
       <nav>
         <ul  class="nav-list">
           <li><a class="nav-link" href="#">Hello</a></li>
           <li><a class="nav-link" href="#">Portfolio</a></li>
           <li><a class="nav-link" href="#">About</a></li>
           <li><a class="nav-link" href="#">Contact</a></li>
         </ul>
+        <div>
         <button class="hamburger">
-          <img src="./assets/icons/ic_menu.svg" alt="menu-icon" />
+          <img class="mobile-app" src="./assets/icons/ic_menu.svg" alt="menu-icon" />
         </button>
       </nav>
     </header>
-    <main>
+    <main class="mobile-app">
       <section class="hero-section hero-section1">
-        <div class="container bg-image">
+        <div class="container bg-image mobile-app">
           <h1 class="primary-heading">Nurgul Kereikhan</h1>
           <p class="paragraph">
             Hello! I am a software developer! I can help you build a product,
             feature or website. Take a look of my works. If you like what you
             see and have a project you need coded, don’t hesitate and contact
             me.
           </p>
-          <button class="cta-btn">Start collaboration</button>
+          <button class="cta-btn mobile-app">Start collaboration</button>
         </div>
+        <div class="mobile-option">
+          <div class="head-dom">
+            <img src="./assets/dom/Group98.png" alt="">
+            <button class="exit">
+              &#x2716;
+            </button>
+          </div>
+        <ul class="mob-style">
+          <li class="dom-bg"><img src="./assets/dom/dots.png" alt=""></li>
+          <li><a class="nav-link" href="#Hello">Hello</a></li>
+          <li><a class="nav-link" href="#Portfolio">Portfolio</a></li>
+          <li><a class="nav-link" href="#About">About</a></li>
+          <li><a class="nav-link" href="#Contact">Contact</a></li>
+          <li class="dom-bg-1"><img src="./assets/dom/dots.png" alt=""></li>
+      </ul>
```

Comment:
- [ ] Please make sure that when the user clicks on any of the nav links, the menu is closed and the page is scrolled to the part the user clicked on. (The page doesn't have any elements that matches the given `href`)
- [ ] [OPTIONAL] Kindly make sure that the background images stay in the background. In other words, background images should be given to containers as background images instead of adding `img` tags.